development-docs/DEV_GUIDE.md

```diff
@@ -382,20 +382,33 @@ the commit if errors are detected:
 ./tools/git-hooks/checkstyle-pre-commit
 ```
 
-## Building container images for other platforms with Docker `buildx`
+## Building container images for other platforms
 
+* build with default `build` command
+When building Strimzi images, it'll add some binaries into the images based on the target platform. The default 
+binaries will be loaded with `linux/amd64` platform compatible. If you want to change it, you can set the environment
+variable `DOCKER_BUILD_ARGS` to pass additional build options such as the platform and set the `TARGETPLATFORM` argument. 
+For example, following commands can be used to build Strimzi images for Linux Arm64 / AArch64 platform:
+
+```
+export DOCKER_BUILD_ARGS="--platform linux/arm64 --build-arg TARGETPLATFORM=linux/arm64"
+make all
+```
+
+* build with Docker `buildx`
 Docker supports building images for different platforms using the `docker buildx` command. If you want to use it to
 build Strimzi images, you can just set the environment variable `DOCKER_BUILDX` to `buildx`, set the environment
-variable `DOCKER_BUILD_ARGS` to pass additional build options such as the platform and run the build. For example
-following can be used to build Strimzi images for Linux on Arm64 / AArch64:
+variable `DOCKER_BUILD_ARGS` to pass additional build options such as the platform and run the build. For example, 
+following can be used to build Strimzi images for Linux Amd64 platform using Arm64 / AArch64 CPU:
 
 ```
 export DOCKER_BUILDX=buildx
+# No need to add `TARGETPLATFORM` build arg since it's default to `linux/amd64`
 export DOCKER_BUILD_ARGS="--platform linux/amd64 --load"
 make all
 ```
```

Comment:
If it is not needed to use special command anymore (it seems not as I don't use it either), should we simply remove it? Does it even still work?